godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -38,6 +53,46 @@ class TractViewController: BaseViewController {
         let activityController = UIActivityViewController(activityItems: [String.localizedStringWithFormat("tract_share_message".localized, "www.knowgod.com")], applicationActivities: nil)
         present(activityController, animated: true, completion: nil)
     }
+    
+    func getData() -> Dictionary<String, Any> {
+        let paragraphContent = [
```

Comment:
This is some dummy data, but, when parsing the XML we should get this structure.